src/components/figures/chart.ts

```diff
@@ -70,7 +70,11 @@ export class ChartFigure extends Component<Props, SpreadsheetEnv> {
   private position = useAbsolutePosition();
 
   get canvasStyle() {
-    return `background-color: ${this.state.background}`;
+    /**
+     * Add sizes to avoid chart out of figure boundaries
+     * when changing background color
+     */
+    return `background-color: ${this.state.background};width: 100%;height: 100%;`;
```

Comment:
IMHO I think it's simplier to add classes `w-100 h-100` in the template. 
`<canvas class="w-100 h-100" t-att-style="canvasStyle" t-ref="graphContainer"/>`

Reply:
@pro-odoo I didn't find a way to test it, especially when we use classes to set height and weight. We cannot get width and height via `getComputedStyle()`.

Reply:
but you can definitely test taht the class w-100 and h-100 are set in the classList :)

```javascript
querySelector("canvas").classList.contains("w-100")
querySelector("canvas").classList.contains("h-100")
``` something like that